docassemble/USCISApplications/data/questions/combined_application_questions.yml

```
@@ -594,7 +594,7 @@ fields:
     required: false
   - What is the expiration date of your most recent passport or travel document?: users[i].passport_expiration_date
     datatype: date
-    required: false      
+    show if: users[i].has_passport
```

```suggestion
    required: False
```
This question is supposed to appear if they have a different document than a passport as well.